WcaOnRails/spec/factories/persons.rb

```diff
@@ -28,7 +28,9 @@
 
     factory :person_with_multiple_sub_ids do
       after(:create) do |person|
-        person.update_using_sub_id!(name: "new #{person.name}")
+        name = person.name
+        person.update!(name: "old name")
+        person.update_using_sub_id!(name: name)
```

Comment:
Yes! It's for the merge_people_spec.rb "requires person1 not have multiple subIds" spec. The problem is that the merge people script requires that the 2 people have the exact same name and citizenship before you attempt to merge them. This change to the person factory allows you to create a person whose "current person"'s name is the name you passed in. Previously, it would create a person whose "current person" is named "new name", and the old person would be named "name"